src/main/java/org/broadinstitute/hellbender/cmdline/argumentcollections/SequenceDictionaryValidationArgumentCollection.java

```diff
@@ -0,0 +1,42 @@
+package org.broadinstitute.hellbender.cmdline.argumentcollections;
+
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+
+/**
+ * interface for argument collections that control
+ */
+public interface SequenceDictionaryValidationArgumentCollection {
+
+    /**
+     * Should sequence dictionary validation be performed
+     * @return true if the tool should perform sequence dictionary validation
+     */
+    boolean performSequenceDictionaryValidation();
+
+
+    /**
+     * most tools will want to use this, it defaults to performing sequence dictionary validation but provides the option
+     * to disable it
+     */
+    class StandardValidationCollection implements SequenceDictionaryValidationArgumentCollection {
```

Comment:
I like having these as inner classes so we don't have separate files for them. I assume that its the case that an inner class inside of an interface is static wrt the containing/implementing class ? Can/should these be static ?

Reply:
They're static by default in an interface and intelli complains that it's redundnat if you mark them as static.  I can add it in though if you think it's clearer.

Reply:
No need to change anything. I just didn't recall seeing that before, but thats about what I suspected.